infinite/types.ts

```diff
@@ -6,13 +6,13 @@ export type InfiniteFetcher<
   Data = any,
   KeyLoader extends InfiniteKeyLoader = InfiniteKeyLoader
 > = KeyLoader extends (...args: any[]) => any
-  ? ReturnType<KeyLoader> extends (
+  ? ReturnType<KeyLoader> extends
       | readonly [...infer K]
       | null
       | false
-      | undefined)
-    ? ((...args: [...K]) => FetcherResponse<Data>)
```

Comment:
I wonder if we should keep the parentheses? @shuding wdyt

Reply:
Yeah it’s easier to read, let’s keep it!

Reply:
Seems they'll be remove anyway, no options for parentheses here but only in arrow functions..